### DISCUSSION TITLE
Archiving and dst drive TBW

### DISCUSSION THREAD

**Discussion by XIII39** [ericaltendorf/plotman#438] [Q&A]
If I understand Archiving correctly then the plots go something like this?

tmp --> dst --> rsyncd path

I'm just thinking about the life of the hdd I'm using as my dst drive in this scenario.
Is it possible to skip the dst drive and go directly from the tmp --> rsyncd path?

**Comment by altendky** [comment]
Sure, set your dst the same as your tmp.  That's how I have it setup.  Presently you have to do this 'manually' in the config by specifying the path twice.  As such you can't do it with multiple tmp drives.  There is a PR in progress to make it doable with multiple tmp drives.

**Reply by XIII39** [reply]
I see.
I am running multiple tmp drives, I'm wondering if it makes sense to either use a single tmp drive or if I can set the dst to the ultimate dst drive as a network share (i.e. dst = smb://harvester.local/plotdrive). I would lose the beauty of plotman looking for free space and not sure if this is the best practice so as I type this it seems that it's not the best idea.

I'm not running any number of plots that will exhaust the TWB of the 'temporary' dst drive all that quickly so I can wait until there is the ability to use multiple tmp drives --> rsyncd but, the longer I can prolong a drives life, the better. 

I will continue to tinker around. 
Thanks!

**Reply by altendky** [reply]
I spent some time today working on that PR and am running it myself now.  If you are willing to test it, that would be helpful.

https://github.com/ericaltendorf/plotman/pull/234

You can install it using the regular installation instructions from the readme but with the URL `git+https://github.com/altendky/plotman@to_push_tmp_dir`.  I have duplicated it in my repo just to avoid linking everyone to a 'third party' repo that neither Eric nor I control.

And yes, losing the archive functionality would not be good.  `:]`

**Reply by XIII39** [reply]
Great! I finally had an opportunity to install it.
I'm going to be running 2 NVMe drives, archiving the plots. 

Quick question. If I have 2 plotters and I want to archive to a central storage (harvester) can I;

`hosts allow = plotter1.ip.address, plotter2.ip.address`

or would I have to call a separate module in the `rsyncd.conf` for each attached plotter the call the appropriate module on the plotter side? 

```
# rsync module name
[chia-1]
.
.
hosts allow = plotter1.ip.address

# rsync module name
[chia-2]
.
.
hosts allow = plotter2.ip.address
```
 I guess I could test this myself pretty quickly, it would probably take me just as long to test it for myself as it did to type this post... Ha!

**Comment by Ed-Yang** [comment]
FYI.  I run a local rsync daemon to use rsync to archive plot to direct connected USB HDD.
